src/plugins/midi-piano/piano-component.js

```diff
@@ -0,0 +1,115 @@
+import PropTypes from 'prop-types';
+import { Piano } from 'react-piano';
```

Comment:
I don't see where "react-piano" comes from, it's not in the `package.json` 🤔

Reply:
That's strange, it's in line 97 of my package.json:  "react-piano": "^3.1.3".

Reply:
I see, it's just in your local code, not part of the code you created the pull request with 😬
<img width="1410" alt="Screen Shot 2022-10-10 at 09 41 10" src="https://user-images.githubusercontent.com/8189923/194819065-2b00ed84-3451-403e-bacc-3dfb17fbb4a4.png">

Reply:
Ok, mystery solved, I was not aware that this package is already imported because of this other plugin that we've not touched in a long time 🤦‍♀️ 
Sorry, I overlooked it.